cmd/traffic_crashlog/traffic_crashlog.cc

```diff
@@ -91,7 +92,8 @@ main(int /* argc ATS_UNUSED */, const char **argv)
   crashlog_target target;
   pid_t parent = getppid();
 
-  diags = new Diags("" /* tags */, "" /* actions */, stderr);
+  BaseLogFile *base_log_file = new BaseLogFile("stderr");
```

Comment:
Why declare `base_log_file`? Just pass `new BaseLogFIle("stderr")` directly to `Diags` constructor.